### ISSUE TITLE
Test connection show "Connection refused" when connection toggle is disconected.

### ISSUE THREAD

**Issue by Transisto** [sparrowwallet/sparrow#70]
Server selection is grayed out when talking with a server.

Test server doesn't try to connect when connection toggle is off.

```
Could not connect:

Connection refused: connect
```

**Comment by Transisto** [comment]
![image](https://user-images.githubusercontent.com/388716/110186347-ef8ce000-7de2-11eb-9d1d-af46dfe69e9f.png)

**Comment by 2pac1** [comment]
Try leaving at blockstream, and go to screen where you have bottom right toggle button, then disable it and then enable it and see if it connects.
I notices inconsistency with Test server functionality.

**Comment by craigraw** [comment]
> I notices inconsistency with Test server functionality.

Yes - recently fixed a bug where the previous server details weren't cleared after a failed test.

**Comment by 2pac1** [comment]
Ok awesome.
